package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "neostandard",
   "version": "0.2.0",
-  "description": "A modern successor to standard",
+  "description": "A modern equivalent to standard",
```

Comment:
I personally like the term "successor", once we land this PR I think the flags offered here and the other modes of operation make this an improvement over the more minimal `standard` feature set.

Reply:
Yeah, this has been improved in `main`